openprescribing/pipeline/management/commands/fetch_and_import_ncso_concessions.py

```diff
@@ -225,6 +226,25 @@ def match_concession_vmpp_ids(items, vmpp_id_to_name):
     return matched
 
 
+def read_concessions_csv(path):
+    def convert(row):
```

Comment:
I'd be minded not to make this an inner function but just a regular top level function instead (`convert_csv_row` or something). One reason is that having is an inner function makes it awkward to test without having to construct an entire CSV file to do it.